### PR TITLE
[POAE7-1420] code refactor to fix targetExpr info lost

### PR DESCRIPTION
**Pull request by yma11**
@Intel-bigdata/bdf-cider-engine @winningsix @jikunshang, please take a review.

### CODE REVIEW

File: velox/exec/HybridExecOperator.cpp [hunk 1]
```diff
@@ -19,7 +19,19 @@
...
+    return {{}, SortAlgorithm::Default, 0, 0};
+  } else {
+    return {
+        ctx->orderByCollation_,
+        SortAlgorithm::SpeculativeTopN,
+        ctx->limit_,
+        ctx->offset_};
+  }
+}
+} //namespace
```
Comment (winningsix) [inline velox/exec/HybridExecOperator.cpp:34]: small hint: line break.

File: velox/exec/HybridExecOperator.cpp [hunk 2]
```diff
@@ -63,4 +75,42 @@ void HybridExecOperator::process() {
   result_ = std::move(input_);
 
 }
-} // namespace facebook::velox::exec
\ No newline at end of file
+
+std::shared_ptr<RelAlgExecutionUnit> HybridExecOperator::getExeUnitBasedOnContext(
```
Comment (winningsix) [inline velox/exec/HybridExecOperator.cpp:79]: Let's move the method of RelAlgExecutionUnit generator into CiderParamContext which is more suitable.

**Review by winningsix — APPROVED** [review]
Overall LGTM. Just some minor comments.

File: velox/core/HybridPlanNode.h [hunk 3]
```diff
@@ -15,9 +15,8 @@
  */
 #pragma once
 
-#include "velox/core/PlanNode.h"
-
 #include "QueryEngine/RelAlgExecutionUnit.h"
+#include "velox/core/PlanNode.h"
```
Comment (jikunshang) [inline velox/core/HybridPlanNode.h:19]: order will influence compile. cc @ZJie1

**Pull request merged**